lib/models/addon.js

```diff
@@ -47,6 +50,18 @@ function deprecatedAddonFilters(addon, name, insteadUse, fn) {
   };
 }
 
+function processModulesOnly(tree) {
+  return new Babel(tree, {
+    modules: 'amdStrict',
+    moduleIds: true,
+    whitelist: ['es6.modules']
```

Comment:
do we need to provide the [amd-name-resolver](https://github.com/chadhietala/amd-name-resolver)? As this doesn't seem to get the config from 146